fuse/greenwood_pd.py

```diff
@@ -59,8 +44,8 @@ def fuse_events(cprr, post_pprr):
 
 
 if __name__ == "__main__":
-    cprr = prepare_cprr()
-    post = prepare_post()
+    cprr = pd.read_csv(data_file_path('clean/cprr_greenwood_pd_2015_2020.csv'))
```

Comment:
Does this cprr have the correct agency text?

Reply:
Corrected.